java/android/nnstreamer/src/main/jni/Android-tensorflow-lite.mk

```diff
@@ -133,7 +133,7 @@ LOCAL_SRC_FILES := $(NNSTREAMER_FILTER_TFLITE_SRCS)
 LOCAL_CXXFLAGS := -O3 -fPIC -frtti -fexceptions $(NNS_API_FLAGS) $(TFLITE_FLAGS)
 LOCAL_C_INCLUDES := $(TF_LITE_INCLUDES) $(NNSTREAMER_INCLUDES) $(GST_HEADERS_COMMON)
 LOCAL_EXPORT_LDLIBS := $(TFLITE_EXPORT_LDLIBS)
-LOCAL_WHOLE_STATIC_LIBRARIES := tensorflow-lite-lib cpufeatures
+LOCAL_WHOLE_STATIC_LIBRARIES := tensorflow-lite-lib
```

Comment:
Can you check below dependency later?
https://github.com/nnstreamer/nnstreamer/blob/42b797f74e5c901c3c471ae04b4c65b96a6f920d/jni/Android.mk#L237

Reply:
@jaeyun-jung Oops, I removed this commit (removing cpufeatures dep). I found that tflite < v1.15.0 requires `cpu-features.h`. PTAL.